reana_server/scheduler/consumer.py

```diff
@@ -125,18 +133,50 @@ def get_consumers(self, Consumer, channel):
             )
         ]
 
-    def on_message(self, workflow_submission, message):
+    def _fail_workflow(self, workflow_id: str, logs: str = "") -> None:
+        self.workflow_status_publisher.publish_workflow_status(
+            workflow_id, status=RunStatus.failed.value, logs=logs,
+        )
+
+    def _retry_workflow_submission(
```

Comment:
Extremely minor: perhaps naming this function `requeue_workflow()` as we [used to have](https://github.com/reanahub/reana-server/blob/f0d55e16ba01ae4e4ca53b823eaa56d6bd2572aa/reana_server/scheduler.py#L132) would be a bit shorter?

Reply:
Why do you think the longer name is less clean?

`requeue` implies it will re-send only, so I would still prefer `retry`.
But, now I look at it, the `workflow` part might be unnecessary. What do you think about `retry_submission()`?